containers/dotnetcore-mssql/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "C# (.NET Core) and MS SQL",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/bash",
+		"mssql.connections": [
+			{
+				"server": "localhost,61433",
```

Comment:
The correct port here is actually 1433. Did you do this due to GitHub Codespaces?  If so, that's because of a bug there.